appeals/web/src/server/appeals/appeal-details/allocation-details/allocation-details.controller.js

```diff
@@ -30,7 +30,8 @@ const renderAllocationDetailsLevels = async (request, response, errors = null) =
 			appealDetails.appealId === request.session.appealId
 				? request.session.allocationLevel
 				: undefined,
-			appealDetails
+			appealDetails,
+			errors && typeof errors === 'object' ? errors['allocation-level'].msg : undefined
```

Comment:
There's a utility for checking whether a value is an object, `isObjectLiteral` in `appeals/web/src/server/lib/object-utilities.js` - not sure if that would work here?